skins/niculskin/index.html.tmpl

```diff
@@ -99,6 +113,18 @@
                         <tr><td>$niculskinLabels.conditions.barometer_min</td><td>$day.barometer.min</td><td>$day.barometer.mintime</td></tr>
                         <tr><td>$niculskinLabels.conditions.rain_sum</td><td>$day.rain.sum</td><td></td></tr>
                         <tr><td>$niculskinLabels.conditions.rain_rate_max</td><td>$day.rainRate.max</td><td>$day.rainRate.maxtime</td></tr>
+                        #if $day.UV.has_data
+                        <tr><td>High UV</td><td>$day.UV.max</td><td>$day.UV.maxtime</td></tr>
```

Comment:
The same with the labels here. Add them to the skin.conf file and just refer to them from the template, e.g. `$niculskinLabels.conditions.uv_max`.